XDMA/linux-kernel/xdma/libxdma.c

```diff
@@ -3530,6 +3536,28 @@ ssize_t xdma_xfer_submit(void *dev_hndl, int channel, bool write, u64 ep_addr,
 
 		switch (xfer->state) {
 		case TRANSFER_STATE_COMPLETED:
+                        if (write == 0) {
```

Comment:
`write` is boolean. `if(!write)` is more readable and more fit
```suggestion
                        if (!write) {
```